inscriptions/ISic000593.xml

```diff
@@ -185,11 +190,17 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AA">
+                <ab>
+                    <lb n="1"/><name><expan><abbr>D</abbr></expan></name> <g ref="#interpunct">·</g> <name>Sempron</name><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="2"/><name><expan><abbr>D</abbr><ex>ecimo</ex></expan></name> <g ref="#interpunct">·</g> <name>Iunio</name> <g ref="#interpunct">·</g> <expan><abbr>D</abbr><ex>ecimo</ex></expan> <expan><abbr>l</abbr><ex>iberto</ex></expan>
+                    <lb n="3"/><name>Optato</name>
+                    <lb n="4"/><name>Sempronia</name> <g ref="#interpunct">·</g> <name><expan><abbr>D</abbr></expan></name> <gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
I think D must be simply orig, because it is not obviously a name after Sempronia (unless D.L. or D.F.)

Reply:
I think I was too confident on the D.L, or D.F.